backend/app/rest/api/rest_private_test.go

```diff
@@ -503,10 +504,18 @@ func TestRest_AnonVote(t *testing.T) {
 	assert.Equal(t, map[string]bool(nil), cr.Votes)
 }
 
+type FS struct {}
```

Comment:
Same as in other place, please rename to something like `mockFS` and put at the end of the file, and get rid of `log` package usage.